tools/fast_puller_.py

```diff
@@ -45,6 +45,13 @@
 parser.add_argument('--directory', action='store',
                     help='Where to save the image\'s files.')
 
+parser.add_argument('--certificates', nargs='*', help='A comma separated ' +
```

Comment:
Maybe this parser argument can get pulled into a common helper since it's duplicated at the moment.

Reply:
Look at what jmillikin did here for logging: https://github.com/google/containerregistry/blob/master/tools/logging_setup_.py#L31